doc/source/user_guide/options.rst

```diff
@@ -487,6 +487,8 @@ styler.sparse.index                     True         "Sparsify" MultiIndex displ
                                                      elements in outer levels within groups).
 styler.sparse.columns                   True         "Sparsify" MultiIndex display for columns
                                                      in Styler output.
+styler.max.elements                     262144       Maximum number of datapoints that Styler will render
```

Comment:
styler.display.max_elements ? i think something like that is more instructive no?

Reply:
or 
styler.render.max_elements (as all things are display based)

Reply:
agreed, even maybe `styler.max_elements`, but changed to yours